calcom/cal.com
# fix: city names were not showing in the options.

## What does this PR do?

While selecting timezones city options were not shown, this is being solved with this pr.

- Fixes #17067
- Fixes CAL-4503



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

checkout any timezone drop down in the app, you should be getting options after searching for a city.

- Are there environment variables that should be set?NO
- What are the minimal test data to have?N/A
- What is expected (happy path) to have (input and output)? to see cities as options in timezones dropdowns
- Any other important info that could help to test that PR N/A


## Images 
![brave_screenshot_localhost](https://github.com/user-attachments/assets/ae41e34f-1d0e-4803-8395-94317291ec93)
